src/utils/ruleCalls.js

```diff
@@ -34,22 +36,19 @@ export const getChangelog = async ruleId =>
     .get(`${process.env.API_URL}/change_logs/${ruleId}`)
     .catch(err => err.response);
 
-export const getActivityLog = async () =>
-  axios.get(`${process.env.API_URL}/change_logs/`).catch(err => err.response);
-
-export const getFilteredActivityLog = async ({
-  limit,
-  offset,
-  filterBy,
-  option,
-}) =>
-  axios
+export const getActivityLog = async ({ limit, offset, filterBy, option }) => {
+  let filters = "";
+  if (option) {
+    filters = `&filterby=${filterBy}&option=${option}`;
```

Comment:
Do these filters behave okay even if filterBy and option are empty strings? &filterby=&option= or some combination of them being empty still work? it could be worth setting defaults or commenting about the filters if those cases will error out on you and to comment about how they must be filled out or not in the query params

Reply:
Hmm i'm not sure I understand what you mean by this :o  

Currently, the filters do work if option is empty, and the user has no way to make filterBy empty since it's all dropdowns. If option is left empty, it'll ignore filterBy&option and send a request to get everything